431682: openldap: ldapmodify segfaults on request with NOOP control

Source: upsteam, cvs diff -r 1.157 -r 1.158 ldapmodify.c
Reviewed-By: Jan Safranek <jsafrane@redhat.com>

The uninitialized varibale cased ldapmodify client to crash, when any 
'-e' parameter was used.

Index: ldapmodify.c
===================================================================
RCS file: /repo/OpenLDAP/pkg/ldap/clients/tools/ldapmodify.c,v
retrieving revision 1.157
retrieving revision 1.158
diff -u -r1.157 -r1.158
--- clients/tools/ldapmodify.c	5 Apr 2004 13:40:07 -0000	1.157
+++ clients/tools/ldapmodify.c	25 Jul 2004 21:05:29 -0000	1.158
@@ -316,7 +316,7 @@
 		)
 	{
 		int err;
-		int i;
+		int i = 0;
 		LDAPControl c[1];
 
 #ifdef LDAP_GROUP_TRANSACTION
